src/common/components/input-error/input-error.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
I guess we can extend from this for specific input checks? Like valid email addresses, valid BTC addresses, etc.?

Reply:
yes we will use for each input this input-error